hathor/transaction/storage/transaction_storage.py

```diff
@@ -541,6 +558,41 @@ def get_all_sorted_txs(self, timestamp: int, count: int, offset: int) -> Transac
         """
         raise NotImplementedError
 
+    def add_value(self, key: str, value: str) -> None:
```

Comment:
Why must the value be an `str`? Can't we save using json or pickle?

Reply:
The idea was to simplify the first implementation. I think we should support `Any` type but rocksdb for instance requires only bytes, so depending on the type it's not just a direct conversion. 

That's why I started with `str` only because it covers what we need for now and many cases, then we can expand later.

Reply:
Ok. We can improve it later since it is not a requirement right now.